seed/views/v3/properties.py

```diff
@@ -143,9 +143,11 @@ def search(self, request):
         # this is the entrypoint to the filtering backend
         # https://www.django-rest-framework.org/api-guide/filtering/#custom-generic-filtering
         qs = self.filter_queryset(qs)
-        # using list-comprehension here instead of Serializer(qs, many=True) b/c
-        # it was not properly serializing the objects with that method. No idea why
-        return JsonResponse([PropertyViewAsStateSerializer(x).data for x in qs.all()], safe=False)
+        # converting QuerySet to list b/c serializer will only use column list profile this way
+        return JsonResponse(
+            PropertyViewAsStateSerializer(list(qs), context={'request': request}, many=True).data,
```

Comment:
Nice - I don't remember us making recent changes to the `PropertyViewAsStateSerializer`, so I'm a bit surprised that it was working without `{'request': request}`. Not concerned about it though!